gsa/src/web/graphql/alerts.js

```diff
@@ -121,14 +129,30 @@ export const DELETE_ALERTS_BY_IDS = gql`
   }
 `;
 
-export const TEST_ALERT = gql`
-  mutation testAlert($id: UUID!) {
-    testAlert(id: $id) {
+export const DELETE_ALERTS_BY_FILTER = gql`
+  mutation deleteAlertsByFilter($filterString: String!) {
+    deleteAlertsByFilter(filterString: $filterString) {
       ok
     }
   }
 `;
 
+export const EXPORT_ALERTS_BY_FILTER = gql`
+  mutation exportAlertsByFilter($filterString: String) {
+    exportAlertsByFilter(filterString: $filterString) {
+      exportedEntities
```

Comment:
Wouldn't it be better to call that property `exportedEntityIDs` because it just returns a list of IDs?

Reply:
exportedEntities? That's supposed return raw xml. Maybe I'm misunderstanding something big here...